src/linux/backend-v4l2.cpp

```diff
@@ -2116,9 +2124,16 @@ namespace librealsense
                 auto buffer = _md_buffers[buf.index];
                 buf_mgr.handle_buffer(e_metadata_buf, _md_fd, buf, buffer);
 
-                // pushing metadata buffer to syncer
-                _video_md_syncer.push_metadata({std::make_shared<v4l2_buffer>(buf), _md_fd, buf.index});
-                buf_mgr.handle_buffer(e_metadata_buf, -1);
+                if (buf.flags & V4L2_BUF_FLAG_ERROR)
+                {
+                    LOG_DEBUG_V4L("METADATA - VIDIOC_DQBUF flags with flag = " << std::hex << buf.flags <<", index = " << std::dec << buf.index << " for fd " << _md_fd << " seq " << buf.sequence);
```

Comment:
Same as previous comment about a `failed..` addition in the message

Reply:
Agreed